pinc/page_tally.inc

```diff
@@ -32,19 +32,21 @@ foreach ( $Round_for_round_id_ as $round )
 
 // -----------------------------------------------------------------------------
 
-// The following variables assume that the site has been configured with an
-// "Entry-Level Round" (ELR): a single round where new users start out and
-// become familiar with the site.  The page-tallies from this round are
-// distinguished from others in that they are (deemed to be) a fair indicator
-// of the user's experience.
-
-$users_ELR_page_tallyboard = new TallyBoard( $ELR_round->id, 'U' );
-$teams_ELR_page_tallyboard = new TallyBoard( $ELR_round->id, 'T' );
-
-// See TallyBoard.inc for explanation of these two strings:
-//
-list($joined_with_user_ELR_page_tallies,$user_ELR_page_tally_column) =
-    $users_ELR_page_tallyboard->get_sql_joinery_for_current_tallies( 'u_id' );
+function get_ELR_tallyboards()
+{
+    global $ELR_round;
+
+    // The following variables assume that the site has been configured with an
+    // "Entry-Level Round" (ELR): a single round where new users start out and
+    // become familiar with the site.  The page-tallies from this round are
+    // distinguished from others in that they are (deemed to be) a fair indicator
+    // of the user's experience.
+
```

Comment:
Would it be useful to add a comment that `U` is for users and `T` is for teams?

Reply:
Indeed.